packages/big-design/src/components/Modal/Modal.tsx

```diff
@@ -49,6 +50,7 @@ const ModalHeader: React.FC<ModalHeaderProps> = ({ children, withBorder }) => {
 
 export class Modal extends React.PureComponent<ModalProps> {
   static defaultProps: Partial<ModalProps> = {
+    appendBodyOverflow: true,
```

Comment:
Can you think of a use case where we wouldn't want to lock the scroll? We chatted about making it an opt-out but I'm not sure anymore. What do you think about always locking the page scroll and we can add an opt-out prop in the future if we get the use case? 🤔